qlty-check/src/parser/reek.rs

```diff
@@ -0,0 +1,98 @@
+use super::Parser;
+use anyhow::Result;
+use qlty_types::analysis::v1::{Category, Issue, Level, Location, Range};
+use serde::{Deserialize, Serialize};
+
+pub struct Reek {}
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct ReekJson {
```

Comment:
The name `ReekJson` signals to me that this struct will represent the entire JSON structure, however this seems like this corresponds with a specific smell so I would call this `ReekSmell`. 

Also, does it need to be `pub`?